codegen/build-oas.sh

```diff
@@ -21,9 +33,9 @@ update_apis_repo() {
 update_templates_repo() {
 	echo "Updating templates repo"
 	pushd codegen/python-oas-templates
-		git fetch
-		git checkout main
-		git pull
+		# git fetch
```

Comment:
OOC why'd you remove these?

Reply:
Nice catch, I didn't mean to leave these commented. I disabled the pull because I was experimenting with some local changes to get the datetime parsing properly.